### ISSUE TITLE
Fix `People` navigation point

### ISSUE THREAD

**Issue by lislis** [rubycorns/rorganize.it#467]
This is the current markup for the `People` navigation point:

``` html
<li class="active">
  <a href="#" data-toggle="dropdown" class="dropdown-toggle">People <b class="caret"</b></a>
  <ul class="dropdown-menu" role="menu">
      <li><a href="/people">All</a></li>
      <li><a href="/coaches">Workshop Coaches</a> </li>
  </ul>
</li>
```

It's very awkward to try and click on `All`.  I'd like to suggest the following changes:
- make the dropdown appear on `:hover`
- just clicking `People` will actually redirect to `/people` (currently `href="#"` not good)
- `All` will also redirect to `/people`
- `Workshop Coaches` stays the same
- all `<a>` tags should have a `title=""` attribute. This is good practice for links as it provides more context and improves usability and accessibility

**Comment by katrin-k** [comment]
Checklist as above: 

- [ ] make the dropdown appear on :hover
- [ ] just clicking People will actually redirect to /people (currently href="#" not good)
- [ ] All will also redirect to /people
- [x] all <a> tags should have a title="" attribute. (Done for header a item only at this point.)

**Comment by katrin-k** [comment]
I've added commits for this in branch `467-fix-people-navigation`. 

However, the dropdown event handler is driving me nuts. One way to make the dropdown appear on hover + assign the href /people was to remove the attribute `data-toggle="dropdown"` from the HMTL. => This completely removes any link to the Bootstrap JS.

Problem 1: It does that on all viewport sizes => the people-dropdown menu is not usable on mobile anymore.

My attempts at only removing data-toggle for a `window.width > 979px` resolves the mobile issue, but brings:

Problem 2: Bootstrap puts the event handler and, thus, the data-toggle attribute back in place once the People-menu is clicked.

The current commit `51f0b85` has the JQuery in `assets/javascripts`. Any frontenders to the rescue? cc @lislis

**Comment by lislis** [comment]
@katrin-k maybe just throw away all the bootstrap markup and build a custom dropdown with just css? It's fun, simple and effective! Plus no bootstrap! Let me know if you need more specifics, I can prep something

**Comment by katrin-k** [comment]
@lislis aaah… right… you did say something in the line of getting rid of bootstrap. 
pure css it is then :)

**Comment by katrin-k** [comment]
I'm gonna close this issue. 
Design/style aspects are invalid due to redesign. 
Accessibility is solved in redesign.

**Comment by til** [comment]
> all <a> tags should have a title="" attribute. This is good practice for links as it provides more context and improves usability and accessibility

Hm, while fixing specs in the Redesign branch we stumbled over slightly differing texts in the links in the toplevel navigation, e.g. "People" and title="All people". We fixed that (for now) by removing the title tags, but maybe that was an overreaction - if you say they should be put back we can do that. How useful (for accessibility) are titles for links really, when the link text is already fully descriptive of what the link does? I would imagine that a screen reader is totally able to read the link text. Of course it's a different story for links that are only an image.
